src/main/scala/com/datastax/sparkstress/WriteTask.scala

```diff
@@ -42,7 +56,9 @@ abstract class WriteTask[rowType](
     printf("Done Setting up CQL Keyspace/Table\n")
   }
 
-  def getKeyspaceCql(ksName: String): String = s"CREATE KEYSPACE IF NOT EXISTS $ksName WITH replication = {'class': 'NetworkTopologyStrategy', 'Analytics': ${config.replicationFactor} }"
+  def getKeyspaceCql(ksName: String, localDC: String): String =
+    s"""CREATE KEYSPACE IF NOT EXISTS $ksName
+       |WITH replication = {'class': 'NetworkTopologyStrategy', '$localDC': ${config.replicationFactor} }""".stripMargin
```

Comment:
we need the same change for streaming task please

Reply:
of course!